tests/console/firewall_enabled.pm

```diff
@@ -21,10 +21,15 @@ use warnings;
 use testapi;
 use version_utils "is_upgrade";
 use Utils::Architectures 'is_ppc64le';
+use service_check;
 
 sub run {
     my ($self) = @_;
-    if ($self->firewall eq 'firewalld') {
+    my $firewall = $self->firewall;
+    # We need to check firewalld not SuSEfirewall2, if we have done
+    # SuSEfirewall2 to firewalld service check.
+    $firewall = 'firewalld' if (($firewall eq 'SuSEfirewall2') && (exists $default_services->{susefirewall}->{service_check_func}));
```

Comment:
In fact, I think this only thinking about our migration regression test, not work for other test such opensuse and non migration regression test.

Reply:
Offline talked with Huajian, it is fine for me.

Reply:
Offline talked with Lemon and all concerns resolved.